codebuild/builder.py

```diff
@@ -516,23 +649,27 @@ def _run_command(command):
         build_dir = tempfile.mkdtemp()
     _log_command(['mkdir', build_dir])
```

Comment:
Log first

Reply:
Can't here, don't have the directory name until mkdtemp() returns.